src/Table/index.scss

```diff
@@ -0,0 +1,71 @@
+.nds-table {
+  display: grid;
+  grid-auto-rows: auto;
+  row-gap: 0;
+}
+
+.nds-table-body,
+.nds-table-header {
+  display: grid;
+  grid-auto-rows: subgrid;
+  grid-template-columns: subgrid;
+  grid-column: 1 / -1; // participate in all parent columns
```

Comment:
In CSS grid properties, declaring grid tracks works similarly to python list comprehension.

This is saying "I want the body and header grids to span from the first to the last column of the parent grid, regardless of how many there are". `-1` == last